### PR TITLE
use @codait/max-image-segmenter

### PR DESCRIPTION
**Pull request by vabarbosa**
@kastentx here are the changes for using `@codait/max-image-segmenter`. let me if you run into any issues or have any questions or want additional changes. thanks.

### CODE REVIEW

**Comment by kastentx** [comment]
Looks good for the most part. The performance is great!

Could we squelch these messages though?

```
loading model...
model load: 42.425ms
running model...
model inference: 790.218ms
running model...
model inference: 760.138ms
```

I'm also getting this warning, but I think it might go away once we merge:
```
objc[1044]: Class GNotificationCenterDelegate is implemented in both /Users/nick/Code/MY_FORKS/magicat/node_modules/canvas/build/Release/libgio-2.0.0.dylib (0x108523578) and /Users/nick/Code/MY_FORKS/magicat/node_modules/@codait/max-image-segmenter/node_modules/canvas/build/Release/libgio-2.0.0.dylib (0x109782498). One of the two will be used. Which one is undefined.
```

**Comment by vabarbosa** [comment]
@kastentx good catch, i missed that. extraneous logging has been removed. thanks.

File: package.json [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-    "model"
+    "app.js"
   ],
   "bin": {
     "magicat": "app.js"
   },
   "dependencies": {
-    "@tensorflow/tfjs": "^0.13.4",
-    "@tensorflow/tfjs-node": "^0.1.20",
+    "@codait/max-image-segmenter": "^0.1.4",
```
Comment (kastentx) [inline package.json]: We'll need to update this to `v0.1.7` I think. I'm still getting the version with logs.

Comment (vabarbosa) [reply package.json]: done

**Pull request merged**